### ISSUE TITLE
Deprecate this crate?

### ISSUE THREAD

**Issue by JohnTitor** [rust-lang/rust-semverver#373]
I've passively maintained this crate for the libc CI, but it doesn't use it anymore and this crate now has some issues as below:
- this crate depends on the save-analysis feature, which is planned to remove in the near future
- this crate doesn't work nicely on some code

Both need some hard work (in particular, the former would need to do rearchitecting), but I'm afraid that I don't have time to do it currently for other projects' maintenance.
Because there are now some alternatives in the community, like https://github.com/obi1kenobi/cargo-semver-checks or https://github.com/Enselic/cargo-public-api, I think it's time to deprecate this crate.

@ibabushkin @Xanewok Any thoughts on this?

**Comment by obi1kenobi** [comment]
I'd be happy to help users of this crate migrate to cargo-semver-checks, if that's something they'd like to do. Do you happen to know which projects other than libc use semverver right now?

**Comment by JohnTitor** [comment]
Great!
> Do you happen to know which projects other than libc use semverver right now?

I'm not aware, but probably it'd be a good way to add instructions on README, I think.

**Comment by Xanewok** [comment]
I agree, we should deprecate this. The save-analysis feature is now removed from the compiler. Thank you @JohnTitor for your work maintaining this repository!

**Comment by JohnTitor** [comment]
I attempted to publish a new version with a deprecation notice but the team on the rust-lang-nursery was removed, it seems: https://crates.io/teams/github:rust-lang-nursery:semver-owners, https://github.com/orgs/rust-lang-nursery/teams
We have to revive the team as the crate has that team only as an owner.

@Mark-Simulacrum Could you create the `semver-owners` team on that org temporarily and publish this crate? Once it's done, it'd be great if you could archive this repository too (I don't have admin access here).
